### PR TITLE
Update kotlin journey-test file with java track's

### PR DESCRIPTION
**Pull request by Smarticles101**
Reference [java#866](https://github.com/exercism/java/pull/688)

### CODE REVIEW

**Comment by stkent** [comment]
I'm going to retry this to verify it's a real issue 🤷‍♂️

**Comment by Smarticles101** [comment]
Totally forgot to add the updated Travis config. I'll add that file as soon as I can

**Comment by stkent** [comment]
Ah, hopefully that's the ticket!

**Comment by Smarticles101** [comment]
Okay that should be better, I think the build failed because I didn't `chmod +x` the journey-test file

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@
 language: java
 jdk:
   - oraclejdk8
+env:
+  - SCRIPT=bin/unit-tests.sh
+  - SCIRPT=bin/journey-test.sh
```
Comment (stkent) [inline .travis.yml]: @Smarticles101 `SCIRPT` -> `SCRIPT` :)

**Comment by stkent** [comment]
@Smarticles101 makes sense; I just spotted a small typo that prevented the journey tests from running at all (comment inline).

**Comment by Smarticles101** [comment]
Got it :+1:

**Comment by stkent** [comment]
LGTM!

**Pull request merged**